### PR TITLE
docs(manuals): add policy docs for managers

### PR DESCRIPTION
**Pull request by fgerschwiler**

### CODE REVIEW

File: docs/docs/manuals/admin-policies.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+title: ZITADEL Policies
+---
+
+Policies are configurations of all the different parts of the IAM. For all parts we have a suitable default in the IAM.
+The default configuration can be overridden for each organization.
+
+## General
+
+You can find this settings in the menu organisation in the section polcies.
```
Comment (livio-a) [inline docs/docs/manuals/admin-policies.md:10]:
```suggestion
You can find these settings in the menu organisation in the section polcies.
```

File: docs/docs/manuals/admin-policies.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+Each policy can be overriden and resetted to the default.
+
+## Password Complexity
+
+With the password complexity policy you can define how a password should look.
```
Comment (livio-a) [inline docs/docs/manuals/admin-policies.md:15]:
```suggestion
With the password complexity policy you can define the requirements for a users password.
```

File: docs/docs/manuals/admin-policies.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+| Force MFA | Force a user to register and use a multifactor authentication |
+| Passwordless | Choose if passwordless login is allowed or not |
+
+![Login Policy](/img/manuals/policies/console_org_login.png)
+
+### Multifactors / Second Factors
+
+In the multifactors section you can configure what kind of multifactors should be allowed.
+Multifactors: 
+- U2F (Universal Second Factor) with PIN
```
Comment (livio-a) [inline docs/docs/manuals/admin-policies.md:44]: we should not that this is required for passwordless

File: docs/docs/manuals/admin-policies.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+Secondfacrots:
```
Comment (livio-a) [inline docs/docs/manuals/admin-policies.md:46]:
```suggestion
Secondfactors: 
```

File: docs/docs/manuals/admin-policies.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+Secondfacrots: 
+- OTP (One Time Password)
+- U2F (Universal Second Factor)
+
+![Second- and Multifactors](/img/manuals/policies/console_org_second_and_multi_factors.png)
+
+### Identity Providers
+
+You can configure all kinds of external identity providers for identity brokering, which support OIDC (OpenIDConnect).
```
Comment (livio-a) [inline docs/docs/manuals/admin-policies.md:54]:
```suggestion
You can configure all kinds of external identity providers for identity brokering, which support OIDC (OpenID Connect).
```

File: docs/docs/manuals/admin-policies.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+Make a new identity provider configuration and enable it in the list after.
```
Comment (livio-a) [inline docs/docs/manuals/admin-policies.md:55]:
```suggestion
Create a new identity provider configuration and enable it in the list afterwards.
```

File: docs/docs/manuals/admin-policies.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+You can configure a light and a dark design.
+
+Make sure you click the "Set preview as current configuration" button after you finish your configuration. Before this it will only be set as your preview configuration.
+
+| Setting | Description |
+| --- | --- |
+| Logo | Upload your logo for the light and the dark design. |
+| Colors | You can set four different colors to design your login page and email. (Background-, Primary-, Warn- and Font Color) |
+| Font | Upload your custom font |
+| Force MFA | Force a user to register and use a multifactor authentication |
```
Comment (livio-a) [inline docs/docs/manuals/admin-policies.md:72]:
```suggestion
```

File: docs/docs/manuals/admin-policies.md [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+| Hide Loginname suffix | If you enable "hide loginname suffix" your loginname suffix (Domain) will not be shown in the login page |
```
Comment (livio-a) [inline docs/docs/manuals/admin-policies.md:73]:
```suggestion
| Hide Loginname suffix | If enabled,  your loginname suffix (Domain) will not be shown in the login page |
```

**Review by livio-a — CHANGES_REQUESTED** [review]

File: docs/docs/manuals/admin-policies.md [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+| Username Password allowed | Possibility to login with username and password |
+| External IDP allowed | Possibility to login with an external identity (e.g Google, Microsoft, Apple, etc)|
+| Force MFA | Force a user to register and use a multifactor authentication |
+| Passwordless | Choose if passwordless login is allowed or not |
+
+![Login Policy](/img/manuals/policies/console_org_login.png)
+
+### Multifactors / Second Factors
+
+In the multifactors section you can configure what kind of multifactors should be allowed.
```
Comment (fgerschwiler) [inline docs/docs/manuals/admin-policies.md:42]:
```suggestion
In the multifactors section you can configure what kind of multifactors should be allowed. For passwordless U2F (Universial Second Factor) with PIN is required. There is no other option.
```

Comment (livio-a) [reply docs/docs/manuals/admin-policies.md:42]: In the multifactors section you can configure what kind of multifactors should be allowed. For passwordless to work, it's required to enable U2F (Universial Second Factor) with PIN. There is no other option at the moment.

**Review by livio-a — APPROVED** [review]

**Pull request merged**